### PR TITLE
feat(amf): Minor clean up in ambr calculation

### PR DESCRIPTION
**Pull request by panyogesh**
Fix:
  1. Storing the AMBR calculation in smf_ctxt when received
     from AMF
  2. Using the same for the PDUSessionAggregateMaximumBitRate
  3. Correcting the DL bandwidth update from SMF

Test:
  1. Basic Unit Testing
  2. PDU Session Establishment with Simulator

Signed-off-by: Yogesh Pandey <yogesh@wavelabs.ai>

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: lte/gateway/c/session_manager/SessionStateEnforcer.cpp [hunk 1]
```diff
@@ -651,7 +651,7 @@ void SessionStateEnforcer::prepare_response_to_access(
   rsp->mutable_session_ambr()->set_max_bandwidth_dl(
       config.rat_specific_context.m5gsm_session_context()
           .default_ambr()
-          .max_bandwidth_ul());
+          .max_bandwidth_dl());
```
Comment (themarwhal) [inline lte/gateway/c/session_manager/SessionStateEnforcer.cpp:654]: Was this a bug? Maybe we should have a unit test to cover these conversions?

**Review by themarwhal — APPROVED** [review]
sessiond changes look good to me! (see small note about potentially adding unit test coverage for the modified function)

File: lte/gateway/c/session_manager/SessionStateEnforcer.cpp (hunk 1 above)
Comment (panyogesh) [reply lte/gateway/c/session_manager/SessionStateEnforcer.cpp:654]: Sure.. Will add the testcase

**Comment by uri200** [comment]
Hey @panyogesh  is this fixed somewhere else? should we close this PR?

**Comment by panyogesh** [comment]
Fixed as part of #10821

**Pull request closed without merging**